partials/query.editor.html

```diff
@@ -1,13 +1,23 @@
-<query-editor-row query-ctrl="ctrl" has-text-edit-mode="true">
+<!--
+<query-editor-row query-ctrl="ctrl" class="blueflood-datasource-query-row">
+    <div class="gf-form-inline">
+        <div class="gf-form">
+            <metric-segment-model property="ctrl.target.target" get-options="ctrl.getOptions()" on-change="ctrl.onChangeInternal()" css-class="tight-form-item-xxlarge"></metric-segment-model>
+        </div>
+    </div>
+</query-editor-row>-->
+
+
+<query-editor-row query-ctrl="ctrl" has-text-edit-mode="true" class="blueflood-datasource-query-row">
 
     <div class="gf-form" ng-show="ctrl.target.textEditor">
-        <input type="text" class="gf-form-input" ng-model="ctrl.target.target" spellcheck="false" ng-blur="ctrl.refresh()"></input>
+        <input type="text" class="gf-form-input" ng-model="ctrl.target.target" spellcheck="false" ng-blur="ctrl.refresh()"/>
     </div>
 
     <div ng-hide="ctrl.target.textEditor">
         <div class="gf-form-inline">
             <div ng-repeat="segment in ctrl.segments" role="menuitem" class="gf-form">
-                <metric-segment segment="segment" get-options="ctrl.getAltSegments($index)" on-change="ctrl.segmentValueChanged(segment, $index)"></metric-segment>
+                <metric-segment property="ctrl.target.target" segment="segment" get-options="ctrl.getAltSegments($index)" on-change="ctrl.segmentValueChanged(segment, $index)"></metric-segment>
```

Comment:
Correction: I see this change in the original src code. maybe the src and this transpiled code  were not in sync when I added them to rackerlabs. Make sure these changes won't blow anything and you still can see the metrics otherwise lgtm.